### DISCUSSION TITLE
请问点评集可以设置“只显示xx字数”吗？

### DISCUSSION THREAD

**Discussion by ShengDai123** [WatsonYao/nian_app#113] [Q&A] [answered]
我对每一本书籍的点评经常可以写到一千字以上，每次想快速浏览点评集里的书籍标题时，都要翻很久，因为我的点评实在太长太占屏幕了。
请问有没有办法设置一下呢？既可以选择“显示完整点评”，也可以选择“只显示点评的前100字或者X百字”，点进去才显示完整点评。这样想翻阅标题的时候也很快速。谢谢！🙏🏻

**Comment by WatsonYao** [comment, marked as answer]
实验室有个点评集折叠文本 可以试试

**Reply by ShengDai123** [reply]
谢谢 已解决
